rtgui/options.cc

```diff
@@ -2236,7 +2239,7 @@ int Options::saveToFile (Glib::ustring fname)
     if (f == nullptr) {
         std::cout << "Warning! Unable to save your preferences to: " << fname << std::endl;
 #ifndef RAWTHERAPEE_CLI
-        Glib::ustring msg_ = Glib::ustring::compose(M("MAIN_MSG_WRITEFAILED"), fname.c_str());
+        Glib::ustring msg_ = Glib::ustring::compose (M ("MAIN_MSG_WRITEFAILED"), fname.c_str());
```

Comment:
@Beep6581 I don't get this one: Why isn't the call operator for `fname.c_str` prepended with a space but the others are?